radicle-node/src/main.rs

```diff
@@ -10,6 +10,19 @@ use radicle_node::prelude::{Address, NodeId};
 use radicle_node::Runtime;
 use radicle_node::{logger, service};
 
+pub const HELP_MSG: &str = r#"
+Usage
+
+   radicle-node [<option>...]
+
+Options
+
+    --connect          <peer>    Connect to the given peer address on start
+    --external-address <address> Publicly accessible address
+    --help                       Print help
+    --listen           <address> Address to listen on
```

Comment:
Can you add a bit more spacing before the descriptions?
```suggestion
    --listen           <address>    Address to listen on
```